src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * 
+ */
+package seedu.addressbook.data.person;
+
+/**
+ * @author Darius
+ *
+ */
+public interface Printable {
+	/**
+	  * Returns a concatenated version of the printable strings of each object.
+	  */
+	public String getPrintableString(Printable... printables);
```

Comment:
As mentioned, this interface's single method is 
public String getPrintableString( );   //no parameter